docs/content/en/docs/how-tos/testers/_index.md

```diff
@@ -6,4 +6,25 @@ weight: 15
 
 This page discusses how to set up Skaffold to run container structure tests after building an artifact.
 
-{{% todo 1076 %}}
+Container structure tests are consistency checks for containers.
+Skaffold relies on [container-structure-test](https://github.com/GoogleContainerTools/container-structure-test) to execute those tests, and requires its [binary](https://github.com/GoogleContainerTools/container-structure-test/releases) to be installed.
+
+When configured for a container image, the structure tests are run after every container build.
```

Comment:
This line is a bit confusing, maybe

Container structure tests are defined per image in the skaffold config. Every time an artifact is rebuilt, skaffold will run the associated structure tests on that image. If the tests fail, skaffold will not continue on to the deploy stage.

WDYT?

Reply:
That's perfect. I totally missed that failing tests would stop the deployment. Good catch!